indico/modules/events/registration/forms.py

```diff
@@ -60,6 +60,9 @@ class RegistrationFormForm(IndicoForm):
                               filters=[lambda x: x if x is not None else 0],
                               widget=NumberInput(step='0.01'),
                               description=_("A fixed fee all users have to pay when registering."))
+    notification_sender_address = StringField(_('Notification sender address'), [IndicoEmail()],
+                                              description=_('Email address set as the sender of all notifications '
+                                                            'sent to users'))
```

Comment:
`. If empty, <insert noreply address here> is used`

You'll probably want to set the description in `__init__` to be able to hav a dynamic part in it.